ansible/roles/datadog/templates/datadog.conf.j2

```diff
@@ -18,3 +18,7 @@ use_mount: no
 {% if inventory_host in groups['proxy'] %}
 dogstreams: {{ log_home }}/{{ deploy_env }}-timing.log:{{ datadog_parsers_dest }}/nginx/timings:parse_nginx_timings
 {% endif %}
+
+{% if inventory_hostname in groups['webworkers'] %}
```

Comment:
if we do add them it'd be nice if we could also add the machine or service type to the log format